test/badge.spec.js

```diff
@@ -34,6 +34,7 @@ describe('Badge Test', function() {
         clearDownloadDirectory();
         // Set download directory path 
         let options = new chrome.Options();
+        options.addArguments("--lang=de-DE");
```

Comment:
Is this still necessary? It seems you changed the German names to English ones either way